app/controllers/admin/getting_started_guide_controller.rb

```diff
@@ -0,0 +1,68 @@
+class Admin::GettingStartedGuideController < ApplicationController
+
+  before_filter :ensure_is_admin
+
+  rescue_from ReactOnRails::PrerenderError do |err|
+    Rails.logger.error(err.message)
+    Rails.logger.error(err.backtrace.join("\n"))
+    redirect_to root_path, flash: { error: "Error prerendering in react_on_rails. See server logs." }
```

Comment:
This error message doesn't mean much to a normal user. Prerender error handling is actually probably something we need to think about a bit more, also in my branch.